src/styles/abstracts/_mixin.scss

```diff
@@ -0,0 +1,47 @@
+@mixin flexCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+// MEDIA QUERY MANAGER
+/*
+0 - 600px:      Phone
+600 - 900px:    Tablet portrait
+900 - 1200px:   Tablet landscape
+[1200 - 1800] is where our normal styles apply
+1800px + :      Big desktop
+$breakpoint argument choices:
+- phone
+- tab-port
+- tab-land
+- big-desktop
+ORDER: Base + typography > general layout + grid > page layout > components
+1em = 16px
+*/
```

Comment:
When multiple individuals are working on a project, having references like this can be quite helpful. In this particular case, I have implemented a mixin that utilizes the em unit for media queries. While it may not be necessary if I am working on this project alone, Please let me know if you have any further questions or concerns. Thank you.

Reply:
Why I agree on the importance of references in big projects, I don't think references should be in `*.scss` file. Can we move them to the `README.md` file or better if we can create a directory named `/documentation` where we will manage all technical requirements such as this.

Reply:
@rud-wtag Any update on moving the text to some other place?

Reply:
@hhu-wtag bhai, somehow I missed this conversation tomorrow. And I will do it by today.